glitter/pages/admin.py

```diff
@@ -20,19 +20,45 @@
 from glitter.admin import GlitterAdminMixin
 from glitter.models import Version
 
-from .forms import DuplicatePageForm
+from .forms import DuplicatePageForm, PageAdminForm
 from .models import Page
 
 
+def get_page_admin_fields():
```

Comment:
There's quite a bit of repetition here with this function, fieldsets, and get_fields.

Could you remove this function, remove fieldsets, remove fields, and create a `get_fieldsets` method instead?